v_3_2_4/master_template.go

```diff
@@ -1837,22 +1837,6 @@ write_files:
         user: scheduler
       name: service-account-context
     current-context: service-account-context
-
-- path: /etc/kubernetes/encryption/k8s-encryption-config.yaml
-  owner: root
-  permissions: 600
-  content: |
-    kind: EncryptionConfig
-    apiVersion: v1
-    resources:
-      - resources:
-        - secrets
-        providers:
-        - aescbc:
-            keys:
-            - name: key1
-              secret: {{ .ApiserverEncryptionKey }}
-        - identity: {}
```

Comment:
We want that for azure and kvm, don't we?

Reply:
Oh shit. I have to check the other operators how this is supposed to be. My thinking is that they have to inject it on their own. The template change here is fine since this should not be there because the management is different across operators. I need to get into KVM and Azure though. Maybe you can point out some things for Azure?

Reply:
In general we have `ExtraManifests` which you can set in both azure and kvm operators https://github.com/giantswarm/k8scloudconfig/blob/392865b96d922615f7b4b946962ffd13e18a9983/v_3_2_4/types.go#L30

But I'd be more keen on adding `DisableEnscryptionAtREST` like we do with calico to keep manifests in sync https://github.com/giantswarm/k8scloudconfig/blob/392865b96d922615f7b4b946962ffd13e18a9983/v_3_2_4/types.go#L14

Reply:
I did a thing. Please have another look. Currently working on the operators. Will ping there after testing confirmation.